appender/src/test/java/com/vlkan/log4j2/redis/appender/RedisServerExtension.java

```diff
@@ -30,14 +30,15 @@ class RedisServerExtension implements BeforeEachCallback, AfterEachCallback {
 
     private final RedisServer redisServer;
 
-    RedisServerExtension(int port, String password) {
+    RedisServerExtension(int port, String username, String password) {
         this.port = port;
         try {
             this.redisServer = RedisServer
                     .builder()
                     .port(port)
                     .bind("0.0.0.0")
-                    .setting("requirepass " + password)
+                    .setting("requirepass default-user-pass")
+                    .setting("user " + username + " on -DEBUG +@all ~* >" + password)
```

Comment:
`requirepass` is not needed anymore:
```suggestion
                    .setting("user " + username + " on -debug +@all ~* >" + password)
```